packages/squiggle-lang/__tests__/ReducerInterface/ReducerInterface_Distribution_test.res

```diff
@@ -45,10 +45,30 @@ describe("eval on distribution functions", () => {
   describe("add", () => {
     testEval("add(normal(5,2), normal(10,2))", "Ok(Normal(15,2.8284271247461903))")
     testEval("add(normal(5,2), lognormal(10,2))", "Ok(Sample Set Distribution)")
-    testEval("add(normal(5,2), 3)", "Ok(Point Set Distribution)")
-    testEval("add(3, normal(5,2))", "Ok(Point Set Distribution)")
-    testEval("3+normal(5,2)", "Ok(Point Set Distribution)")
-    testEval("normal(5,2)+3", "Ok(Point Set Distribution)")
+    testEval("add(normal(5,2), 3)", "Ok(Normal(8,2))")
+    testEval("add(3, normal(5,2))", "Ok(Normal(8,2))")
+    testEval("3+normal(5,2)", "Ok(Normal(8,2))")
+    testEval("normal(5,2)+3", "Ok(Normal(8,2))")
+  })
+  describe("subtract", () => {
+    testEval("10 - normal(5, 1)", "Ok(Normal(5,1))")
+    testEval("normal(5, 1) - 10", "Ok(Normal(-5,1))")
+  })
+  describe("multiply", () => {
+    testEval("normal(10, 2) * 2", "Ok(Normal(20,4))")
+    testEval("2 * normal(10, 2)", "Ok(Normal(20,4))")
+    testEval("lognormal(5,2) * lognormal(10,2)", "Ok(Lognormal(15,4))")
+    testEval("lognormal(10, 2) * lognormal(5, 2)", "Ok(Lognormal(15,4))")
```

Comment:
Not sure if true, should be Lognormal(15,sqrt(8)).

Reply:
`Lognormal(mu1, sigma1) * Lognormal(mu2, sigma2) = Lognormal(mu1 + mu2, sqrt(sigma1^2 + sigma2^2)) // (!!)`